### ISSUE TITLE
Json Format for java.util.UUID

### ISSUE THREAD

**Issue by supersnager** [spray/spray-json#243]
Hi,
UUID type is used very often in json (json-rpc etc), so i think that it will be good to have formater for UUID as standard method in DefaultJsonProtocol.
I can do pull request with it, but i don't know which place in code will be good for it.
I think that this formatter could be in trait StandardFormats and named UUIDFormat.
Is it good idea?

**Comment by jrudolph** [comment]
Yep, sounds good. Please go ahead with a PR and we can discuss details. (that is, if you are still interested, so sorry for the delay)

**Comment by supersnager** [comment]
Nice, im still interested. I'll back to this topic next week.
